tests/buffer_ut/buffer_ut.c

```diff
@@ -1561,6 +1558,81 @@ BEGIN_TEST_SUITE(Buffer_UnitTests)
         BUFFER_delete(res);
     }
 
+    TEST_FUNCTION(BUFFER_create_with_size_succeeds)
```

Comment:
>TEST_FUNCTION(BUFFER_create_with_size_succeeds) [](start = 4, length = 47)

Missing specs and tags in the code.
Since we're partying together on this code, could you please add them?

BUFFER does have specs/tests code so let's keep the bar up. "When in Rome"